init.lua

```diff
@@ -1,164 +1,37 @@
-local S = minetest.get_translator("crafting_bench")
-
-minetest.register_alias("castle:workbench", "crafting_bench:workbench")
-
-local usage_help = S("The inventory on the left is for raw materials, the inventory on the right holds finished products. The crafting grid in the center defines what recipe this workbench will make use of; place raw materials into it in the crafting pattern corresponding to what you want to build.")
-
-if minetest.get_modpath("hopper") and hopper ~= nil and hopper.add_container ~= nil then
-	usage_help = usage_help .. "\n\n" .. S("This workbench is compatible with hoppers. Hoppers will insert into the raw material inventory and remove items from the finished goods inventory.")
-end
-
-
-local crafting_rate = tonumber(minetest.settings:get("crafting_bench_crafting_rate")) or 5
-
-
-minetest.register_node("crafting_bench:workbench",{
-	description = S("Workbench"),
-	_doc_items_longdesc = S("A workbench that does work for you. Set a crafting recipe and provide raw materials and items will magically craft themselves once every @1 seconds.", crafting_rate),
-	_doc_items_usagehelp = usage_help,
-	tiles = {
-		"crafting_bench_workbench_top.png",
-		"crafting_bench_workbench_bottom.png",
-		"crafting_bench_workbench_side.png",
-		"crafting_bench_workbench_side.png",
-		"crafting_bench_workbench_back.png",
-		"crafting_bench_workbench_front.png"
-		},
-	paramtype2 = "facedir",
-	paramtype = "light",
-	groups = {choppy=2,oddly_breakable_by_hand=2,flammable=2},
-	sounds = default.node_sound_wood_defaults(),
-	drawtype = "normal",
-	on_construct = function ( pos )
-		local meta = minetest.get_meta( pos )
-			meta:set_string( 'formspec',
-			'size[10,10;]' ..
-			default.gui_bg ..
-			default.gui_bg_img ..
-			default.gui_slots ..
-			'label[1,0;'..S('Source Material')..']' ..
-			'list[context;src;1,1;2,4;]' ..
-			'label[4,0;'..S('Recipe to Use')..']' ..
-			'list[context;rec;4,1;3,3;]' ..
-			'label[7.5,0;'..S('Craft Output')..']' ..
-			'list[context;dst;8,1;1,4;]' ..
-			'list[current_player;main;1,6;8,4;]' ..
-			'listring[current_name;dst]'..
-			'listring[current_player;main]'..
-			'listring[current_name;src]'..
-			'listring[current_player;main]')
-		meta:set_string( 'infotext', S('Workbench'))
-		local inv = meta:get_inventory()
-		inv:set_size( 'src', 2 * 4 )
-		inv:set_size( 'rec', 3 * 3 )
-		inv:set_size( 'dst', 1 * 4 )
-	end,
-	can_dig = function(pos,player)
-		local meta = minetest.get_meta(pos);
-		local inv = meta:get_inventory()
-		return inv:is_empty("main")
-	end,
-	on_metadata_inventory_move = function(pos, from_list, from_index, to_list, to_index, count, player)
-		minetest.log("action", player:get_player_name().." moves stuff in workbench at "..minetest.pos_to_string(pos))
+futil.check_version({ year = 2023, month = 01, day = 21 })
```

Comment:
What does this do?

Reply:
it checks that the version of futil that's present has the necessary functionality (namely, `FakeInventory.create_copy(inv)`).

Reply:
Would you mind adding a note for that as a comment above this line?